### ISSUE TITLE
[Feature Request]: 

### ISSUE THREAD

**Issue by Mike63x** [FTBTeam/FTB-Mods-Issues#655]
### Mod

FTB Chunks

### Description

In the FTB Chunks there is a setting for "no wilderness" which prevents players from interacting, placing or building in Chunks that are considered unclaimed / "wilderness". What features would be great to add to this settings to choose what kind of actions this prevents. So maybe a server wants to prevent placing and breaking in unclaimed Chunks but wants to allow interaction (opening doors, chest, pressing buttons, etc..)

### Describe the solution you'd like

Adding three modifications to the no wilderness feature. 

Prevent break true/false
Prevent place true/false
Prevent Interact true/false

### Describe alternatives you've considered

Currently searching but unknown at time of this request

**Comment by desht** [comment]
It's a good idea, but I'm not sure of the value of having separate place/break functionality (it would also be inconsistent with existing protections, which are "block edit" and "block interaction").

**Comment by Mike63x** [comment]
I may have found a solution for interactions. https://github.com/FTBTeam/FTB-Mods-Issues/issues/347#issuecomment-1179616916

**Comment by desht** [comment]
Yes, that will definitely help for cases like Lootr chests.  Allowing players to interact with blocks separately from editing them in the wilderness would still be nice to have though (e.g. if some structure generates with closed doors in the wilderness, you may want to allow players to open the doors without actually being able to destroy the structure).

**Comment by ZEROX7** [comment]
i second that

**Comment by ZEROX7** [comment]
> I may have found a solution for interactions. [#347 (comment)](https://github.com/FTBTeam/FTB-Mods-Issues/issues/347#issuecomment-1179616916)
well waht if you just want tem to be able to open doors in the wildness but not on claimed chunks?
